roles/azimuth_capi_operator/tasks/main.yml

```diff
@@ -72,6 +72,32 @@
   loop_control:
     label: "{{ item.metadata.name }}"
 
+- name: Ensure cluster template tenancy access controls are up to date
+  command: >-
+    kubectl annotate --overwrite
+    clustertemplates.azimuth.stackhpc.com/{{ item[0].metadata.name }}
+    {{ item[1] }}{{-
+      '=' + azimuth_capi_operator_cluster_templates_tenancy_access_controls[item[1]] 
+      if item[1] in azimuth_capi_operator_cluster_templates_tenancy_access_controls
+      else '-'
+    }}
+  loop: >-
+    {{-
+      azimuth_capi_operator_templates_cmd.stdout |
+        from_json |
+        json_query('items') |
+        product(acl_annotation_keys) |
+        list
+    }}
+  loop_control:
+    label: "{{ item[0].metadata.name }} - {{ item[1] }}"
+  vars:
+    acl_annotation_keys:
+    - "acl.azimuth.stackhpc.com/allow-list"
+    - "acl.azimuth.stackhpc.com/deny-list"
+    - "acl.azimuth.stackhpc.com/allow-regex"
+    - "acl.azimuth.stackhpc.com/deny-regex"
```

Comment:
There has to be a better way to do this! 🤣 This is horrendous. Let me have a think.